templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="page-header">
+                <h1>{{ "Forms Administration"|translate }}</h1>
+            </div>
+        </div>
+    </div>
+    <div class="row bottom-pad">
+        <div class="col-xs-12">
+            <button type="submit" name="update" class="btn btn-default btn-save">{{ "Update"|translate }}</button>
+        </div>
+    </div>
+    {% if error %}
+        <div class="alert alert-danger">
+            {{ error|translate }}
+        </div>
+    {% endif %}
+    <div class="row">
+        <div class="col-xs-12">
+
+            <!-- Nav tabs -->
+            <ul class="nav nav-tabs" role="tablist">
+                <li role="presentation" class="active">
+                    <a href="#registered"
+                       aria-controls="home" role="tab"
+                       data-toggle="tab">
+                        {{ "Registered"|translate }}
+                        &nbsp;<span class="badge">{{ registered|length }}</span>
+                    </a>
+                </li>
+                <li role="presentation">
+                    <a href="#unregistered"
+                       aria-controls="profile" role="tab"
+                       data-toggle="tab">
+                        {{ "Unregistered"|translate }}
+                        {# Wrap the badge in an if statement so bootstrap will
+                           hide element if the count is empty
+                         #}
+                        &nbsp;<span class="badge">{% if unregistered|length > 0 %}{{ unregistered|length }}{% endif %}</span>
+                    </a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="registered">
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>&nbsp;</th>
+                            <th>Name</th>
+                            <th>Status</th>
+                            <th colspan="2">&nbsp;</th>
+                            <th>{{ "Priority"|translate }}</th>
+                            <th>{{ "Category"|translate }}</th>
+                            <th>{{ "Nickname"|translate }}</th>
+                            <th>{{ "Access Control"|translate }}</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% for row in registered %}
+                            <tr>
+                                <td class="text-muted"><small>{{ row.id }}</small></td>
+                                <td><strong>{{ row.name }}</strong></td>
+                                <td class="{{ row.statusColor }}">
+                                    <i class="{{ row.statusIcon }}"></i>&nbsp;
+                                    {% if row.status == "Registered" %}
```

Comment:
This points out a good issue, where should not use labels in algorithm. Should separate this into a key that is used in the algorithms (and is not translated; ie row.statusId) and a label (which is translated upstream; ie row.statusLabel). Important to keep these things separate; when we internationalized the codebase back in 2008 or so, there were lots of places we had to do this to fully support internationalization and not muck up the algorithm with translated labels.